DEPENDENCIES

```diff
@@ -202,6 +202,83 @@ Prerequisites for building the OpenDataPlane (ODP) API
    1024MB of memory:
    $ sudo ODP_PKTIO_DPDK_PARAMS="-m 1024" ./test/performance/odp_l2fwd -i 0 -c 1
 
+3.5 Code instrumentation with PAPI library (optional)
```

Comment:
As we are not enabling instrumentation in generic library code, maybe this text should go to example/instrum/README with a note here that we support PAPI instrumentation?

Reply:
I am expecting to have more config time options in the future (symbols visibility, etc.) that will affect building of  generic library code.